package.json

```diff
@@ -4,18 +4,20 @@
   "description": "Fiori Fundamentals is a Design System and HTML/CSS Component Library used to build modern Product User Experiences with the SAP look and feel. Learn more about this project at - http://sap.github.io/fundamental/",
   "main": "index.js",
   "scripts": {
-    "build-doc": "npm run build:config && gulp docs-site && cd docs && bundle install && bundle exec jekyll build --config _config.yml,_config-library.yml && cd ..",
-    "build": "npx gulp build:dist",
+    "build:docs": "npm run build:config && npm run build && npx gulp docs-site && cd docs && bundle install && bundle exec jekyll build --config _config.yml,_config-library.yml && cd ..",
     "build:config": "./scripts/create-library-config.js",
-    "deploy": "gh-pages -d docs",
+    "build": "npx gulp build:dist",
+    "deploy": "gh-pages -d docs/_site",
     "lint:fix": "stylelint './scss/**/*.scss' --fix",
     "lint:pre-commit": "printf \"running pre-commit lint...\"  && npm run lint && printf \"done!\n\"",
     "lint": "stylelint './scss/**/*.scss'",
     "release:create": "create-release",
     "release": "./scripts/publish-release.sh",
-    "start": "npm run build:config && npx gulp default",
+    "start": "npm run build && node test/app.js",
+    "start:docs": "npm run build:config && npx gulp default",
```

Comment:
lets make `npm start` continue to run the docs, as it does in fundamental-react and -ngx

Reply:
I would agree -- more in line with the other fundamental repos.  Then maybe a `start:playground` or just `playground` for the test app.